modules/debts_api/app/controllers/debts_api/v0/digital_disputes_controller.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'debts_api/v0/digital_dispute_submission_service'
+require_relative '../../../../lib/debts_api/v0/digital_dispute_submission_service'
```

Comment:
It seems odd we have to do this. Have you tried bouncing your server with the original code?

I'm cool with it but just wanted to make sure. I've seen this happen before.